pkg/loop/internal/pb/relayer.proto

```diff
@@ -0,0 +1,262 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-relay/pkg/loop";
+
+package loop;
+
+import "google/protobuf/empty.proto";
+
+service PluginRelayer {
+  rpc NewRelayer (NewRelayerRequest) returns (NewRelayerReply) {}
+}
+
+message NewRelayerRequest {
+  string config = 1; // toml (is chain instance config enough?)
+  uint32 keystoreID = 2;
+
+  //TODO prometheus? https://smartcontract-it.atlassian.net/browse/BCF-2075
+}
+
+message NewRelayerReply {
+  uint32 relayerID = 1;
+}
+
+service Keystore {
+  rpc Keys (google.protobuf.Empty) returns (KeysReply) {}
+  rpc Sign (SignRequest) returns (SignReply) {}
+}
+
+message KeysReply {
+  repeated string accounts = 1;
+}
+
+message SignRequest {
+  string account = 1;
```

Comment:
No restrictions imposed by this API. This is a bit of a weird case where the chain-specific relayer uses a chain-specific key, so this can vary by chain family. I expect it to continue to evolve as we implement the others.